### PR TITLE
Add poll support

### PR DESCRIPTION
**Pull request by aliraghebi**
Completed poll properties

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 composer.phar
 composer.lock
 vendor/
-build/
\ No newline at end of file
+build/
+.idea/
```
Comment (irazasyed) [inline .gitignore]: Please remove the `.idea` from this file. It should go in your global gitignore file.

File: src/Objects/Message.php [hunk 2]
```diff
@@ -9,53 +9,53 @@
...
+ * @property Invoice $invoice                (Optional). Message is an invoice for a payment, information about the invoice.
  * @property SuccessfulPayment $successfulPayment      (Optional). Message is a service message about a successful payment, information about the payment.
- * @property string            $connectedWebsite       (Optional). The domain name of the website on which the user has logged in.
- * @property PassportData      $passportData           (Optional). Telegram Passport data
- * @property string            $replyMarkup            (Optional). Inline keyboard attached to the message. login_url buttons are represented as ordinary url buttons.
+ * @property string $connectedWebsite       (Optional). The domain name of the website on which the user has logged in.
+ * @property PassportData $passportData           (Optional). Telegram Passport data
+ * @property string $replyMarkup            (Optional). Inline keyboard attached to the message. login_url buttons are represented as ordinary url buttons.
  */
```
Comment (irazasyed) [inline src/Objects/Message.php]: Please maintain the code styling.

File: src/Objects/Message.php [hunk 3]
```diff
@@ -65,39 +65,40 @@ class Message extends BaseObject
...
+            'poll' => Poll::class,
+            'new_chat_members' => User::class,
+            'left_chat_member' => User::class,
+            'new_chat_photo' => PhotoSize::class,
+            'pinned_message' => self::class,
+            'invoice' => Invoice::class,
             'successful_payment' => SuccessfulPayment::class,
-            'passport'           => PassportData::class,
+            'passport' => PassportData::class,
         ];
```
Comment (irazasyed) [inline src/Objects/Message.php]: Code styling!

**Review by irazasyed — CHANGES_REQUESTED** [review]
Please make the requested changes! Thanks.

**Comment by aliraghebi** [comment]
@irazasyed Solved! Would you please review again?

**Comment by irazasyed** [comment]
Thanks!

**Pull request merged**